packages/components/src/renderers/StorageStatusRenderer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface StorageStatusProps {
+    data: any
+}
+
+export class StorageStatusRenderer extends React.PureComponent<StorageStatusProps, any> {
+
+    render() {
+        const { data } = this.props;
+
+        const value = data && data.get('value');
+
+        if (value.toLowerCase() === 'not in storage') {
+            return data.get('value');
```

Comment:
can we just use the `value` variable here since we've already done the get from data?